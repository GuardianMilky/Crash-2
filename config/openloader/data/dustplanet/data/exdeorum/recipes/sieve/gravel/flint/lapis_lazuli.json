{"type":"exdeorum:sieve","ingredient":{"item":"minecraft:gravel"},"mesh":"exdeorum:flint_mesh","result":"minecraft:lapis_lazuli","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.05}}