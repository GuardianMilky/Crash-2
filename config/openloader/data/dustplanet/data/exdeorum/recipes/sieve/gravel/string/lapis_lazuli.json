{"type":"exdeorum:sieve","ingredient":{"item":"minecraft:gravel"},"mesh":"exdeorum:string_mesh","result":"minecraft:lapis_lazuli","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.03}}