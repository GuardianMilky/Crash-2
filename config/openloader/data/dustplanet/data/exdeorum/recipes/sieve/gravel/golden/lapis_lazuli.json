{"type":"exdeorum:sieve","ingredient":{"item":"minecraft:gravel"},"mesh":"exdeorum:golden_mesh","result":"minecraft:lapis_lazuli","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.1}}